Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream> 
+#include "Person.h"
+#include <string>
+
+std::string n;
+int a;
+
+Person::Person() 
+{
+	name_ = "";
+	age_ = new int(0);
+}
+
+Person::~Person() 
+{
+	free(age_);
```

Comment:
It's better to use delete here. [Here's an article talking about the difference](https://www.geeksforgeeks.org/new-vs-malloc-and-free-vs-delete-in-c/)